### PR TITLE
upload Assignment1

### PR DESCRIPTION
**Pull request by doroteea**

### CODE REVIEW

**Comment by Alex-Cosma** [comment]
Please exclude all folders which contain binary data
`.target` & `.idea` in your case
then push again

thank you!

**Comment by doroteea** [comment]
am incercat sa le sterg? si de pe git si din folder-ul meu de unde am dat commit. Sper ca am si reusit

File: Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Date;
+import java.util.List;
+
+import java.util.*;
+
+public class AdminController {
+    @FXML
+    TextField idTF;
```
Comment (Alex-Cosma) [inline Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java:26]: Not really a controller, is it? Too many view elements

File: Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    @FXML
+    TextArea viewTA;
+
+    @FXML
+    TextArea generateTA;
+
+    private String username,password;
+    private Long id;
+    private Date period;
+    ConnectionStuff con = new ConnectionStuff();
```
Comment (Alex-Cosma) [inline Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java:61]: `ConnectionStuff` is a very poor name!

File: Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    @FXML
+    Button generateB;
+
+    @FXML
+    TextArea viewTA;
+
+    @FXML
+    TextArea generateTA;
+
+    private String username,password;
```
Comment (Alex-Cosma) [inline Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java:58]: I might have never mentioned this, but I will do so now - state of any kind (like this field, users) is never desirable.
Why? you may ask. Well, what if, for example, you would have multiple users logged in and one of them sets this one of these variables to something and another gets to set it to something else. What happens then?
This is why state is not desirable (will talk about this more during the lab)

File: Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+    public void addButtonOnAction(javafx.event.ActionEvent event){
+       try{
+           username = usernameTF.getText();
+           password = passwordTF.getText();
+
+           con.getUserValidator().validate(username, password);
+           final List<String> errors = con.getUserValidator().getErrors();
+           if (errors.isEmpty()) {
+               con.getAuthenticationService().register(username, password);
```
Comment (Alex-Cosma) [inline Assignment 1/Sandor Doroteea/src/main/java/controller/AdminController.java:72]: definitely not ok. You are basically using that "con" as a static object from which you are just using everything. This defies the purpose of dependency injection. It's not about just usage, it's about **injection**

**Pull request merged**